runtime/gc_vlhgc/ContinuationObjectBufferVLHGC.cpp

```diff
@@ -129,9 +133,10 @@ MM_ContinuationObjectBufferVLHGC::iterateAllContinuationObjects(MM_EnvironmentBa
 					omrobjectptr_t object = region->getContinuationObjectList()->getHeadOfList();
 					while (NULL != object) {
 						Assert_MM_true(region->isAddressInRegion(object));
+						envVLHGC->_continuationStats._total += 1;
 						omrobjectptr_t next = extensions->accessBarrier->getContinuationLink(object);
-						J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF((J9VMThread *)env->getLanguageVMThread(), object);
-						if (NULL != continuation) {
+						if (VM_ContinuationHelpers::isActive(*VM_ContinuationHelpers::getContinuationStateAddress((J9VMThread *)env->getLanguageVMThread(), object))) {
```

Comment:
let's break this into 2 lines (fetch the state into a local before the if)

Reply:
... and the same pattern in ContinuationObjectBufferStandard.cpp as well